docs/developer/release/README-release-process.html

```diff
@@ -239,7 +240,42 @@ <h2 id="continuous-integration">Continuous integration tests</h2>
     <li>Daikon: <a href="https://dev.azure.com/mernstdaikon/daikon/_build/latest?definitionId=1&branchName=master"><img src="https://dev.azure.com/mernstdaikon/daikon/_apis/build/status%2Fcodespecs.daikon?branchName=master" alt="codespecs/daikon Azure Pipelines status"></a></li>
   </ul>
 
-  <h2 id="snapshot_release">Snapshot release</h2>
+  <h2 id="snapshot_release">Snapshot release to local Maven repository</h2>
+  <p>This section explains how to release the Maven artifacts to a local Maven repository.</p>
+  <ol>
+    <li>
+      Ensure that the version number in <code>checker-framework/build.gradle</code> (in <code>allprojects { version ... }</code>) ends in <code>-SNAPSHOT</code>.
```

Comment:
For a local repository,  `PublishToMavenLocal` will override whatever is there no matter the version number. `SNAPSHOT` is only required when publishing to the Maven Central snapshot repository.